.github/workflows/publish.yml

```diff
@@ -11,7 +11,7 @@ on:  # yamllint disable-line rule:truthy
       - '.github/**'
     tags:
       - "[0-9]+.[0-9]+.[0-9]+"
-      - "[0-9]+.[0-9]+.[0-9]+-lts"
+      - "[0-9]+.[0-9]+.[0-9]+-(lts|rc)"
```

Comment:
Does this match rc1 etc? If it is a prefix match on the tag then we don't even need the lts|rc line since the first line is sufficient, but if it needs to be a complete match then something is missing.

Or we do a wildcard for anything which starts with [0-9]+.[0-9]+.[0-9]+ using a single line.

Reply:
Yes, it matches `rcn | lts`, but the second line isn't needed. In some cases, we're creating tags that follow the `major.minor.patch` versioning scheme without `rcn | lts`, and asset uploads are required to ensure the correct URL in the ipxe.efi.cfg. To keep things simple, we'll build all tags using the `major.minor.patch` format.